Tiltfile

```diff
@@ -0,0 +1,31 @@
+allow_k8s_contexts('kind-topolvm-e2e')
+
+def get_all_go_files():
+  return str(local("find -type f -name '*.go' -not -name '*_test.go'")).split("\n")
+
+all_go_files = get_all_go_files()
+ignores=["e2e/tmpbin", "e2e/bin", "e2e/build", "build", "bin", "include", "testbin", "e2e/topolvm.img", "*/.docker_temp_*"]
+
+local_resource("hypertopolvm", "make -C e2e e2e-bin", deps=all_go_files, ignore=ignores)
+
+docker_build("topolvm:dev", "e2e/tmpbin", dockerfile="e2e/Dockerfile")
+
+load("ext://cert_manager", "deploy_cert_manager")
```

Comment:
In my opinion, I do not like this extension very much.
Every time tiltfile is loaded, the extension runs a test. It takes a little time.
https://github.com/tilt-dev/tilt-extensions/blob/6f4d3436c557d70bb0810b0da1acb99c364120b6/cert_manager/Tiltfile#L61-L65

I prefer to install cert-manager beforehand.